src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -13,6 +13,7 @@ function cssClass(element: string) {
 
 export interface Props {
   theme: "normal";
+  attachments?: React.ReactNode;
```

Comment:
hmm should this be `React.ReactNode[]`? Or has it been compiled into one node by this point

Reply:
Great catch!!

Reply:
If that's the case, then that applies to the other types of AnnouncementBubble too 👍